charts/astronomer/templates/_helpers.yaml

```diff
@@ -205,11 +205,13 @@ s3:
     secretKeyRef:
       name: {{ template "houston.backendSecret" . }}
       key: connection
+{{- if not (((.Values.houston.config.deployments).database).connection) }}
```

Comment:
A change needed to seperate our houston db from our airflow db. Previously if a customer set Values.houston.config.deployments.database.connection it was ignored